integration_test/Bindings/Tcl/simple.tcl

```diff
@@ -0,0 +1,36 @@
+# REQUIRES: bindings_tcl
+# RUN: tclsh %s -- %TCL_PATH% | FileCheck %s
+load [lindex $argv 1]libcirct-tcl[info sharedlibextension]
+
+set circuit [circt load MLIR [lindex $argv 1]../../../../integration_test/Bindings/Tcl/Inputs/simple.mlir]
```

Comment:
Thanks for moving the example mlir. Super nitpick, but could `../../../../integration_test/Bindings/Tcl/Inputs/simple.mlir` just be `../Inputs/simple.mlir`?

Reply:
I can pass in another argument if you'd like.